sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.16.0 (unreleased)
+## 1.16.0 (2024-04-22)
```

Comment:
Can you revert this? This date is supposed to be when the new SDK version is released.

Reply:
Reverted. Thanks for letting me know.